run_dir/design/barcode.html

```diff
@@ -8,21 +8,27 @@
 
 {% block stuff %}
 <h1><span id="page_title">NGI Plate Label and Barcode Printing</span></h1>
+<br>
+<h3><span>Label and barcode printing for workset plates or internal use</span></h3>
 <p>Here you can print plate labels and barcodes on the label printer "Zebra". Either enter
 a text in the text field or upload a file.<p>
 <p>Valid files are barcode files generated by LIMS or text files. In text files,
   each line will be printed on a separate label.<p>
 <br>
 <div id="query">
-  <form class="form-horizontal" enctype="multipart/form-data" role="form" id="barcode_from_file" method="post" action="/barcode" onsubmit="return submit_form()">
+  <form class="form-horizontal" enctype="multipart/form-data" role="form" id="barcode_from_file" method="post" action="/barcode" onsubmit="return submit_form('lab_labels')">
     <div class="pull-left" style="margin-right:5px">
       <input id="print_label_file" type="file" name="file_to_print" class="btn btn-default" />
+      <input type="hidden" id="formType" name="formType" value="lab_labels">
     </div>
     <br>
     <br>
     <br>
     <div class="pull-left" style="margin-right:5px">
       <input id="print_label_string" size="37" type="text" name="text_to_print" class="btn btn-default" placeholder="your text here" />
+      <td style="font-size:20px;"><strong>&nbsp;&nbsp;&nbsp; print with barcode: &nbsp;</strong></td>
+      <input id='print_with_barcodeHidden' type='hidden' value='noBC' name="print_with_barcode">
```

Comment:
This doesn't seem to be used anywhere.

Reply:
It is actually a way to have a value for the checkbox if it is unticked. I would get an error otherwise. The 'noBC' value is used in barcode.py line 57.